src/lib/expansion-panel/expansion-panel-adapter.ts

```diff
@@ -70,4 +86,44 @@ export class ExpansionPanelAdapter extends BaseAdapter<IExpansionPanelComponent>
   public animationEnd(): void {
     this._innerElement.style.removeProperty('overflow');
   }
+
+  public tryLocateTargetButton(id: string): void {
+    this._targetButton = this._tryFindTargetButton(id);
```

Comment:
We won't need this method any more because we're not storing the element reference and the core file will not need to call this.